index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+    See Project
+</button>
+</div>
+    </div>
+    </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+
+</section>
+</section>
+<section class="about-me">
+  <section class="about-me-page">
+    <div class="about-me-all">
+      <h1 class="about-me-h1">
+        About me
+      </h1>
+      <div class="about-me-para-div">
+      <p class="about-me-para">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="get-resume-div">
+    <button class="get-resume">
+      Get My Resume
+    </button>
+  </div>
+    </div>
+  </section>
+  </section>
+
+  
+  <section class="languages-framework-skills">
+    <ol class="general" id="general">
+      <li class="ol1 ">  
+        Language
+        <ul class="js gen">
```

Comment:
- [ ] Good job on adding skills section but please stick as much as you can to  [Figma desgin](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159077905-ad5d9485-c2a3-4ac3-9245-4242edafa8b4.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159077975-8b571e73-31f9-43ed-bd1a-81df8d03d706.png)

Reply:
@Donard97 Regarding this, I stuck to the www.figma.com to letter fonts and every other, and that was the font and font size that was used but I think you are suggesting I reduce it. Or are you saying otherwise?